version.php

```diff
@@ -30,7 +30,7 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = [25, 0, 7, 1];
+$OC_Version = [25, 0, 7, 2];
```

Comment:
undo this change, bump will follow with https://github.com/nextcloud/server/pull/38754

```suggestion
$OC_Version = [25, 0, 7, 1];
```